.github/workflows/pytest.yml

```diff
@@ -3,13 +3,9 @@ on:
   push:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths:
-      - 'weldx/**'
-      - 'tutorials/**'
-      - '.github/workflows/pytest.yml'
-  # Run every Monday at 6am UTC
```

Comment:
I have removed these conditions here since the current configuration should prevent most of the 'unnecessary' runs anyway and it doesn't pick up project config related changes otherwise @marscher @vhirtham

Reply:
Okay, just need to remember it in case something unforeseen happens in the future 😉

Reply:
it should just run the tests more often/consistently now (which I guess you welcome anyway 🎂 😃 )

Reply:
true... 😏